include/boost/geometry/algorithms/detail/overlay/traversal.hpp

```diff
@@ -354,35 +372,71 @@ struct traversal
         // If both are valid candidates, take the one with minimal remaining
         // distance (important for #mysql_23023665 in buffer).
 
-        // Initialize with 0, automatically assigned on first result
+        turn_operation_type const& op0 = turn.operations[0];
+        turn_operation_type const& op1 = turn.operations[1];
+        signed_size_type const next0 = op0.enriched.get_next_turn_index();
+        signed_size_type const next1 = op1.enriched.get_next_turn_index();
+
+        bool possible[2];
+        possible[0] = traverse_possible(next0);
+        possible[1] = traverse_possible(next1);
```

Comment:
Slightly inconsistent naming. Other variables have indexes at the end of the name instead of being an array. So `possible0`, `possible1`. Or is there a reason for this?

Reply:
I answered earlier: "Good point. Not really, I will harmonize it." (but apparently I did something wrong, it was placed under the wrong comment).
Anyway, there was a reason because it is used for iteration below.
But I now harmonized it (using more arrays of 2 and iterations), will push.
Thank you.